_health-care/_js/components/personal-information/Email.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import ErrorableTextInput from '../form-elements/ErrorableTextInput';
+import { isValidEmail } from '../../utils/validations.js';
+
+/**
+ * Input component for collecting a Email number.
+ *
+ * Validates the input data. Does NOT consider "invalid" email numbers such
+ * as 000-000-0000 to be errors. This is to allow integration testing with
+ * fake data.
+ *
+ * Props:
+ * `error` - Function that returns a string. Only necessary for confirming email.
+ * `label` - String. Not required as a prop, can be passed directly to
```

Comment:
No need to reference ErrorableTextInput int he comment.

To the user of this Component, they shouldn't know that ErrorableTextInput exist. That's an abstraction break.